Release build is breaking on arch 386

On Github Actions is happening:
```
Building for Others / 386 amd64
Number of parallel builds: 1

-->   openbsd/amd64: github.com/quan-to/chevron/cmd/cli
-->   freebsd/amd64: github.com/quan-to/chevron/cmd/cli
-->     windows/386: github.com/quan-to/chevron/cmd/cli
-->   windows/amd64: github.com/quan-to/chevron/cmd/cli
-->     freebsd/386: github.com/quan-to/chevron/cmd/cli
-->    darwin/amd64: github.com/quan-to/chevron/cmd/cli
-->      darwin/386: github.com/quan-to/chevron/cmd/cli
-->     openbsd/386: github.com/quan-to/chevron/cmd/cli

1 errors occurred:
--> darwin/386 error: exit status 2
Stderr: # golang.org/x/sys/unix
Error: /home/runner/go/pkg/mod/golang.org/x/sys@v0.0.0-20210403161142-5e06dd20ab57/unix/syscall_darwin.go:383:45: undefined: Xucred
Error: /home/runner/go/pkg/mod/golang.org/x/sys@v0.0.0-20210403161142-5e06dd20ab57/unix/syscall_darwin.go:384:11: undefined: Xucred
Error: /home/runner/go/pkg/mod/golang.org/x/sys@v0.0.0-20210403161142-5e06dd20ab57/unix/syscall_darwin.go:385:21: undefined: SizeofXucred

Error: Process completed with exit code 1.
```

Locally (MacOs Big Sur 11.2.3) is happening:
```
Building for Linux / arm arm64 386 amd64
#!/bin/bash
Number of parallel builds: 11

-->     linux/amd64: github.com/quan-to/chevron/cmd/cli
-->       linux/arm: github.com/quan-to/chevron/cmd/cli
-->       linux/386: github.com/quan-to/chevron/cmd/cli
-->     linux/arm64: github.com/quan-to/chevron/cmd/cli
Building for Others / 386 amd64
Number of parallel builds: 11

-->    darwin/amd64: github.com/quan-to/chevron/cmd/cli
-->   freebsd/amd64: github.com/quan-to/chevron/cmd/cli
-->      darwin/386: github.com/quan-to/chevron/cmd/cli
-->     freebsd/386: github.com/quan-to/chevron/cmd/cli

1 errors occurred:
--> darwin/386 error: exit status 2
Stderr: cmd/go: unsupported GOOS/GOARCH pair darwin/386
```

--- a/scripts/travis-binary-build.sh
+++ b/scripts/travis-binary-build.sh
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 BUILD_LINUX_ARCH="arm arm64 386 amd64"
-BUILD_OTHER_ARCH="386 amd64"
+BUILD_OTHER_ARCH="amd64"
 BUILD_OS="windows freebsd darwin openbsd"
 
 export GOCACHE=/tmp/gocache

